lib/config/configuration.go

```diff
@@ -195,7 +195,7 @@ func ReadConfigFile(cliConfigPath string) (*FileConfig, error) {
 	if cliConfigPath != "" {
 		configFilePath = cliConfigPath
 		if !utils.FileExists(configFilePath) {
-			return nil, trace.NotFound("file %s is not found", configFilePath)
+			return nil, trace.NotFound("failed to load configuration: configuration file specified '%s' does not exist", configFilePath)
 		}
 	}
 	// default config doesn't exist? quietly return:
```

Comment:
We're now running the same file exists check twice here within the same 10 lines of code - can we refactor this check to a common place?